### PR TITLE
Try out simple-phpunit

### PR DESCRIPTION
**Pull request by greg0ire**
Do not merge, it's just for science.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ lint:
 	git diff --exit-code
 
 test:
-	phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+	vendor/bin/simple-phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
```
Comment (core23) [inline Makefile]: Available since `symfony/phpunit-bridge` 3.2

Comment (greg0ire) [inline Makefile]: Thanks for the help ;)

**Comment by greg0ire** [comment]
I don't understand why it fails now…

**Comment by core23** [comment]
Looks like travis took the same dependencies like https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/198138928, but got a different result. Very strange 😕

**Comment by greg0ire** [comment]
I'm think I understand what's happening here, but not how to fix it, see https://symfony-devs.slack.com/archives/support/p1486155435004685

**Comment by jordisala1991** [comment]
It is better than just using phpunit/phpunit? Just trying to understand

**Comment by greg0ire** [comment]
Yes. See https://symfony-devs.slack.com/archives/random/p1485386338001733

**Comment by greg0ire** [comment]
Yay it finally passes

**Comment by greg0ire** [comment]
Not sure why coverage decreased…

**Comment by core23** [comment]
Great work @greg0ire 🍺

**Comment by greg0ire** [comment]
@core23 I found a better way to do this thanks to @fbourigault : the env var I'm using here explicitely tells `simple-phpunit` NOT to remove prophecy. Coverage increased now lol.

**Comment by greg0ire** [comment]
Performance-wise, the build might be slightly faster, not sure. I think there are little to no implications.

**Comment by greg0ire** [comment]
Out of 3 files, 2 are from dev-kit. The other one is composer.json. If we're going to do this, we will have to require `symfony/phpunit-bridge` 3.2 everywhere first.

**Pull request closed without merging**